include/swift/Serialization/SerializationOptions.h

```diff
@@ -34,12 +34,14 @@ namespace swift {
     StringRef ImportedHeader;
     StringRef ModuleLinkName;
     ArrayRef<std::string> ExtraClangOptions;
+    ArrayRef<std::string> Dependencies;
 
     bool AutolinkForceLoad = false;
     bool EnableNestedTypeLookupTable = false;
     bool SerializeAllSIL = false;
     bool SerializeOptionsForDebugging = false;
     bool IsSIB = false;
+    bool SerializeFileDependencies = false;
```

Comment:
Do we need the flag? Would it be reasonable to rely on the `Dependencies` array being empty?

Reply:
Mm, fair enough. Removing.